[2024-03-21 16:49:08,581] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.compute_lr_score 2024-03-21T16:48:00+00:00 [queued]>
[2024-03-21 16:49:08,614] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: airflow_exam.compute_lr_score 2024-03-21T16:48:00+00:00 [queued]>
[2024-03-21 16:49:08,615] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-03-21 16:49:08,615] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2024-03-21 16:49:08,615] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-03-21 16:49:08,633] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): compute_lr_score> on 2024-03-21T16:48:00+00:00
[2024-03-21 16:49:08,644] {standard_task_runner.py:52} INFO - Started process 14760 to run task
[2024-03-21 16:49:08,652] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', '***_exam', 'compute_lr_score', '2024-03-21T16:48:00+00:00', '--job-id', '9531', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/DAG.py', '--cfg-path', '/tmp/tmp_yb891ah', '--error-file', '/tmp/tmpl6kd0efq']
[2024-03-21 16:49:08,653] {standard_task_runner.py:77} INFO - Job 9531: Subtask compute_lr_score
[2024-03-21 16:49:08,734] {logging_mixin.py:104} INFO - Running <TaskInstance: ***_exam.compute_lr_score 2024-03-21T16:48:00+00:00 [running]> on host 3a3d0401ac26
[2024-03-21 16:49:08,831] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=***_exam
AIRFLOW_CTX_TASK_ID=compute_lr_score
AIRFLOW_CTX_EXECUTION_DATE=2024-03-21T16:48:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-21T16:48:00+00:00
[2024-03-21 16:49:09,286] {python.py:151} INFO - Done. Returned value was: None
[2024-03-21 16:49:09,329] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=***_exam, task_id=compute_lr_score, execution_date=20240321T164800, start_date=20240321T164908, end_date=20240321T164909
[2024-03-21 16:49:09,397] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-03-21 16:49:09,427] {local_task_job.py:151} INFO - Task exited with return code 0
